### DISCUSSION TITLE
Cannot reconnect. Please reload the window

### DISCUSSION THREAD

**Discussion by wjy18666** [coder/code-server#2467] [Q&A]
Hello, I encountered such a problem, can you help me solve it? Chrome running vscode often will Cannot reconnect. Please reload the window pop-up, refreshing is useless, the following is my detailed information:
1. Intranet test
Machine configuration and environment: CentOS7.6 32C64G container runs 4C8G nginx without forwarding
Version: code-server: 3.5.0 vscode: 1.48.2
Execution: code server Chrome browser runs vscode without problems

2. Online
Machine configuration and environment:
Bandwidth 100Mbps, mobile network
Machine 48 core 108G
CentOS7.6
Run vscode in the container.
The container limit is 4C8G, and it is forwarded through 3 layers of nginx

Version: code-server 3.5.0 vscode: 1.48.2

Execution: code server running vscode Chrome browser often Cannot reconnect. Please reload the window pop-up window, it is still the same after refreshing

**Comment by oxy** [comment]
Hey, can you check the configuration for each layer of nginx? I suspect it might be something messing up there - I couldn't reproduce this myself.

**Reply by wjy18666** [reply]
Thank you for your reply. I think this problem may be related to the network and the environment, especially the network is unstable, because the company's small partners redeployed once, and there are few problems. Let's observe it again, thank you very much.

**Comment by sr229** [comment]
Hey there, the reason for this is because the Websocket cannot connect to the client. Please check your nginx/Apache httpd configuration, you may have not allowed websocket connections.
